app/views/sites/api_instructions/_show_web_md.html.haml

```diff
@@ -1,485 +1,77 @@
-  * You can find your site handle on the [Settings](#{edit_site_setting_path(@site)}) page.
   * Replace {YOUR_SEARCH_TERM} with a word or phrase of your choice.
+  * You must use `https`.
 
 
+  ## Parameters
 
-
-  Note that your access key is unique to your site handle so they must be paired properly to return results.
-
-
-
-
-  ## Optional Search Parameters
-
-
-
-
-  All other parameters are optional: `enable_highlighting`, `limit`, `offset`, `sort_by`.
-
-
-
-
-  * ### enable_highlighting
-
-
-
-
-      Enables or disables the highlighting of keywords in the results. The default is 'true' so use 'false' to disable highlighting. The opening and closing highlighting characters are `<U+E000>` and `<U+E001>`, which both look like "". You can learn more about them [here](http://unicodesymbols.wikia.com/wiki/U%2BE000) and [here](http://unicodesymbols.wikia.com/wiki/U%2BE001) (external resources). Your team will determine how to display the characters, whether as bold, italics, or some other preferred highlighting style.
-
-
-
-
-      `#{api_scheme_and_host}/api/v2/search?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}&enable_highlighting=false`
-
-
-
-
-  * ### limit
-
-
-
-
-      Defines the number of results to return. The default is 20, but you can specify between 1 and 50 results.
-
-
-
-
-      `#{api_scheme_and_host}/api/v2/search?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}&limit=5`
-
-
-
-
-  * ### offset
-
-
-
-
-      Defines the number of results you want to skip from the first result. The default is 0 and the maximum is 999.
-
-
-
-
-      `#{api_scheme_and_host}/api/v2/search?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}&offset=20`
-
-
-
-
-  * ### sort_by
-
-
-
-
-      Sort the results by date. The default is sort by relevance.
-
-
-
-
-      `#{api_scheme_and_host}/api/v2/search?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}&sort_by=date`
-
-
-
-
-  ## Response Fields
-
-
-
-
-  Each item returns a unique set of fields.
-
-
-
-
-  * ### web:results
-
-
-
-
-      Web results from our Search.gov indexes.
-
-
-
-
-      | Values           | Description
-      | :--              | :--
-      | title            | Title of the document
-      | url              | URL of the document
-      | snippet          | Summary of the document
-      | publication_date | Publication date of the document (not available on commercial results)
-
-
-
-
-  * ### web:total
-
-
-
-
-      Total number of results available.
-
-
-
-
-  * ### web:next_offset
-
-
-
-
-      Offset for the subsequent results.
-
-
-
-
-  * ### web:spelling_correction
-
-
-
-
-    Spelling correction for your search term.
-
-
+  For descriptions of the following parameters, please see the full documentation on the [OpenGSA Portal](https://open.gsa.gov/api/searchgov-results/).
```

Comment:
I'm presuming that the preceding section got transferred bodily to the OpenGSA portal page referred to; it certainly ought to still be around _somewhere_.